### ISSUE TITLE
Keeping scope in generated routes

### ISSUE THREAD

**Issue by szsoppa** [danschultzer/pow#291]
Hi! Is it possible to generate routes for two scopes?

I'm trying to generate localized and not localized routes this way:

```elixir
  scope "/" do
    pipe_through [:browser]

    pow_routes()
    pow_extension_routes()
    pow_assent_routes()
  end

  scope "/:locale" do
    pipe_through [:browser]

    pow_routes()
    pow_extension_routes()
    pow_assent_routes()
  end
```

However, routes are being generated only for the first scope. Unfortunately, I really need both of these, since for now, it's the only way to meet client requirements. 

Side note: It seems to work for pow_assent.

**Comment by danschultzer** [comment]
You'll need to also name the scope, otherwise they won't be created (to prevent naming collision):

```elixir
  scope "/" do
    pipe_through [:browser]

    pow_routes()
    pow_extension_routes()
    pow_assent_routes()
  end

  scope "/:locale", as: :locale do
    pipe_through [:browser]

    pow_routes()
    pow_extension_routes()
    pow_assent_routes()
  end
```

**Comment by szsoppa** [comment]
Thanks for a quick response @danschultzer 😉

Maybe I'm mistaken, but I assume that generating routes **with and without** scope will create route helpers with a different number of arguments, and therefore there will be no collision on argument level.

That's fine for me, I'm not sure if naming collision is the thing I'm trying to prevent here.

**Comment by danschultzer** [comment]
There will just be a warning, but Pow [actively filters](https://hexdocs.pm/pow/Pow.Phoenix.Router.html#content) the conflicting routes to prevent this: #198

**Comment by danschultzer** [comment]
Getting late here, so I was too fast answering. I think you are right, there should be no collision here, so this is just Pow [filtering the routes](https://github.com/danschultzer/pow/blob/v1.0.13/lib/pow/phoenix/router.ex#L113-L127). I'll look into it tomorrow to see if I can add a check for equal arguments to.

**Comment by danschultzer** [comment]
Solved in #292, let me know if it works for you 😄 

```elixir
{:pow, github: "danschultzer/pow", ref: "filter-routes-bindings"}
```

**Comment by szsoppa** [comment]
That was fast! Thank's @danschultzer, it's working 😉
